lib/Quantization/Quantization.cpp

```diff
@@ -702,15 +701,15 @@ class FunctionQuantizer : public FunctionConverter {
   const ElemKind quantizationPrecisionBias_;
 
 public:
-  /// Creates a function quantizer for \p F using the quantization
-  /// parameters defined by \p quantizationInfos and target quantization
+  /// Creates a function quantizer for \p F using the profiling
+  /// parameters defined by \p profilingInfos and target quantization
   /// precision defined by \p quantizationPrecision.
   /// \p B and \p doNotQuantizeKinds are used to check which nodes shouldn't be
   /// converted. \p assertAllNodesQuantized is used as a debugging tool; if
   /// true then if the backend does not support a node as quantized for the
   /// given \p quantizationPrecision then the program will exit with an error.
   FunctionQuantizer(Function &F, const Backend &B, quantization::Schema schema,
-                    llvm::ArrayRef<NodeQuantizationInfo> quantizationInfos,
+                    const std::vector<NodeProfilingInfo> &profilingInfos,
```

Comment:
Why the change away from ArrayRef? Can generateNodeQuantizationInfos just accept ArrayRef<NodeProfilingInfo>?

Reply:
I changed it back to llvm::ArrayRef.